src/tFPDF/PDF.php

```diff
@@ -892,13 +916,25 @@ public function SetFillColor($int_red, $int_green = null, $int_blue = null)
      * @param null $int_green
      * @param null $int_blue
      */
-    public function SetTextColor($int_red, $int_green = null, $int_blue = null)
+    public function SetTextColor()
     {
         // Set color for text
-        if (($int_red == 0 && $int_green == 0 && $int_blue == 0) || $int_green === null) {
-            $this->str_text_color = sprintf('%.3F g', $int_red / 255);
-        } else {
-            $this->str_text_color = sprintf('%.3F %.3F %.3F rg', $int_red / 255, $int_green / 255, $int_blue / 255);
+        $args = func_get_args();
+        if (count($args) and is_array($args[0])) {
+            $args = $args[0];
+        }
+        switch (count($args)) {
+            case 1:
+                $this->str_text_color = sprintf('%.3f g', $args[0] / 100);
```

Comment:
Your implementation of "g" divides argument value by 100, instead of the old 255. Is that on purpose or is this an oversight?
If it's on purpose do you have any reference links to help me understand why the change was necessary?